src/purchase_listinator/models/internal/shopping_cart.clj

```diff
@@ -31,37 +31,37 @@
    :quantity-changed s/Int})
 
 (s/defschema PurchaseListCategoryDeleted
-  {:id               s/Uuid
-   :moment           s/Num
-   :event-type       (s/eq :purchase-list-category-deleted)
-   :user-id          s/Uuid
-   :shopping-id      s/Uuid
-   :category-id      s/Uuid
-   :purchase-list-id s/Uuid})
+  {:id                           s/Uuid
+   :moment                       s/Num
+   :event-type                   (s/eq :purchase-list-category-deleted)
+   :user-id                      s/Uuid
+   (s/optional-key :shopping-id) s/Uuid
```

Comment:
I suggest to left the shopping-id as optional for these events since the [adapters](https://github.com/gumberss/PurchaseListinator/blob/88e07314047d5b1a667217effe49f20511d5f2e8/src/purchase_listinator/adapters/in/shopping_purchase_list_events.clj) are not providing it, what do you think?